charts/app-config/templates/kubescape.yaml

```diff
@@ -13,13 +13,18 @@ spec:
     repoURL: 'https://github.com/kubescape/helm-charts'
     targetRevision: HEAD
     helm:
-      values: |
+      valueFiles:
+        - values.yaml
+      parameters:
```

Comment:
You can keep the original syntax if you use [valuesObject](https://argo-cd.readthedocs.io/en/stable/user-guide/helm/) instead of `parameters`.

Reply:
I will have a look at that if this fails again